test/project_types/node/commands/create_test.rb

```diff
@@ -52,7 +69,9 @@ def test_check_default_npm_registry_is_production
         FileUtils.touch('test-app/server/handlers/client.js')
         FileUtils.touch('test-app/server/handlers/client.cli.js')
 
+        @context.expects(:capture2e).with('which', 'npm').returns(['/usr/bin/npm', mock(success?: true)])
```

Comment:
I wonder if it's worth extracting out some of the common stubbing since we're repeating it several times?